nion/instrumentation/MultiAcquire.py

```diff
@@ -577,10 +580,9 @@ def acquire_multi_eels_spectrum_image(self):
         finally:
             self.__acquisition_finished_event.set()
             self.acquisition_state_changed_event.fire({'message': 'end', 'description': 'spectrum image'})
+            self.acquisition_state_changed_event.fire({'message': 'end processing'})
             # TODO: configure line repeat
-            self.shift_y(0)
             self.shift_x(0)
             self.adjust_focus(0)
             if hasattr(self, 'scan_parameters'):
-                delattr(self, 'scan_parameters')
-            logging.debug("end")
\ No newline at end of file
+                delattr(self, 'scan_parameters')
```

Comment:
Minor point: can you add a newline at the end of the file?

Reply:
Done.